client/shipping-label/state/actions.js

```diff
@@ -140,21 +140,41 @@ const refreshPreview = ( dispatch, getState, context ) => {
 	} );
 };
 
-export const openPrintingFlow = () => ( dispatch, getState, context ) => {
+export const openPrintingFlow = () => (
+	dispatch,
+	getState,
+	context,
+	getErrors = getFormErrors
```

Comment:
Yeh, I struggled with this one :) @DanReyLop - I tried to stub it, but honestly it was a lot of extra effort and at the end I didn't get it working. Figured better to test the error condition than not, hence the extra param.

@jeffstieler - the aliasing is to avoid the `no-shadow` lint error. Happy to change it to an ignore comment instead, don't have a strong preference on how to make eslint happy here.